pnr/openroad/meta.yaml

```diff
@@ -0,0 +1,77 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('openroad-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: openroad
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - disable-tests.patch
+  - git_url: https://github.com/The-OpenROAD-Project/OpenSTA.git
+    git_rev: master
```

Comment:
is that ok to pull master rather than using the version pinned as a submodules?

Reply:
we don't actually need to do this, since `git_url` already does a recursive clone https://github.com/conda/conda-build/blob/master/conda_build/source.py#L328

Reply:
Done